src/SFA.DAS.EAS.Web/Orchestrators/EmployerAgreementOrchestrator.cs

```diff
@@ -85,7 +85,7 @@ public EmployerAgreementOrchestrator(IMediator mediator, ILog logger, EmployerAp
             return response;
         }
 
-        public async Task<OrchestratorResponse<EmployerAgreementListViewModel>> Get(string hashedId,
+        public virtual async Task<OrchestratorResponse<EmployerAgreementListViewModel>> Get(string hashedId,
```

Comment:
do we need make the method `virtual`?.

Reply:
It looks like there's nothing overriding this method

Reply:
Its for mocking as we don't have interfaces for the orchestrators. The reason why we don't have interfaces for them was mainly political. Many devs didn't like orchestrators as there is a 1-1 relationship with controllers and so they didn't want to create the interfaces to highlight this issue. No completely sure its be useful tbh. I did mention the interfaces issues but was told to carry on not implementing them.